tests/unit/concrete/pool-addresses-provider/constructor/constructor.t.sol

```diff
@@ -20,11 +20,8 @@ contract Constructor_PoolAddressesProvider_Unit_Concrete_Test is PoolAddressesPr
     function test_RevertWhen_GovernorIsZeroAddress() external {
         string memory marketId_ = defaults.MARKET_ID();
 
+        vm.expectRevert(abi.encodeWithSelector(Errors.GovernorZeroAddress.selector));
         isleGlobals.transferGovernor(address(0));
-
-        vm.expectRevert(
-            abi.encodeWithSelector(Errors.PoolAddressesProvider_InvalidGlobals.selector, address(isleGlobals))
```

Comment:
If we can prevent zero address from `IsleGlobals::initialize` or `Governable::transferGovernor` then this `PoolAddressesProvider_InvalidGlobals` case seems will never happen 🤔